README.md

```diff
@@ -189,6 +189,10 @@ To access a private container registry, follow the steps on [this page](http://k
 
 We recommend you use ImagePullSecrets, but if you would like to configure access on the minikube VM you can place the `.dockercfg` in the `/home/docker` directory or the `config.json` in the `/home/docker/.docker` directory.
 
+## Add-ons
+
+In order to have minikube properly start/restart custom addons, place the addon(s) you wish to be launched with minikube in the `.minikube/addons` directory.  Addons in this folder will be moved to the minikubeVM and launched each time minikube is started/restarted.
```

Comment:
We have merged in this change for now.  That is a good suggestion, I will add links to to additional documentation for add ons in a future PR as I imagine many people will test out k8s features using minikube and this could be there starting off point.  Regarding the addons that minkube supports, we should work fine with anything which only requires one node and does not use features that require a Cloud Provider. These include:
LoadBalancers
PersistentVolumes
Ingress